HW_2.py

```diff
@@ -0,0 +1,39 @@
+import random
+import string
+
+def get_the_list():
+    uni_list = []
+    # get the number of key/values in a dict + dicts quantity
+    quantity = random.randint(2,10)
+    for i in range(quantity):
+        # generate lists for keys and values
+        keys = random.sample(string.ascii_lowercase, quantity)
+        values = random.sample(range (0, 100), quantity)
+        # combine them into dict
+        my_dict = dict(zip(keys, values))
+        # append new dict to a list
+        uni_list.append(my_dict)
+    print(uni_list)
+    return uni_list
+
+ def create_common_dict(uni_list):
+    common_dict = {}
+    i = 0
+    for dictionary in uni_list:
+        for key, value in dictionary.items():
+            if key not in common_dict:
```

Comment:
The logic is not completely correct.
For example, the script added first item d=8 to the common_dict, then your next "d=55" value was compared with value in the common_dict, so the script added d1=55 to the common dict. But then the script didn't delete old value from the common_dict. So if you have one more "d" value, it'll compare with old "d=8", not new "d1=55"

![image](https://user-images.githubusercontent.com/34323554/106672070-25f2f780-65b8-11eb-933a-323646013db2.png)

![image](https://user-images.githubusercontent.com/34323554/106718945-624f4380-660a-11eb-96a8-541c8f76cc8e.png)